citestwheel.Dockerfile

```diff
@@ -19,14 +19,25 @@ ARG DEBIAN_FRONTEND=noninteractive
 ENV PYENV_ROOT="/pyenv"
 ENV PATH="/pyenv/bin:/pyenv/shims:$PATH"
 
-RUN apt-get update \
-        && apt-get upgrade -y \
-        && apt-get install -y --no-install-recommends \
-        wget curl git jq ssh \
-        make build-essential libssl-dev zlib1g-dev \
-        libbz2-dev libreadline-dev libsqlite3-dev wget \
-        curl llvm libncursesw5-dev xz-utils tk-dev unzip \
-        libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+RUN <<EOF
+set -e
+if [[ "${LINUX_VER}" =~ "ubuntu" ]]; then
+  echo 'APT::Update::Error-Mode "any";' > /etc/apt/apt.conf.d/warnings-as-errors
+  # update git > 2.17
+  add-apt-repository ppa:git-core/ppa -y
```

Comment:
This depends on `software-properties-common` to my knowledge.

Reply:
ahh damn it.